src/components/mdTable/mdTablePagination.vue

```diff
@@ -48,7 +48,7 @@
         subTotal: 0,
         currentSize: parseInt(this.mdSize, 10),
         currentPage: parseInt(this.mdPage, 10),
-        totalItems: !isNaN(this.mdTotal) && Number.MAX_SAFE_INTEGER
+        totalItems: isNaN(this.mdTotal)?Number.MAX_SAFE_INTEGER:parseInt(this.mdTotal, 10)
```

Comment:
I agree with you! I said it wrong. What I meant was the fix is not the awesome part that you did, but the template too.
Could you fix the template part?